src/routes/user.js

```diff
@@ -25,6 +25,7 @@ const {
 router.get(
   '/:id(\\d+)',
   authenticate(),
+  ensureRole('Administrator'),
```

Comment:
Does this return private fields and that's why it needs an ensureRole? We might want to figure out a way to do the ensureRole check in the method handler and then conditionally return additional information if the user has the right role to see it - just trying to make sure that we can call as many of these routes as possible as a normal user to get the data we will eventually need on the frontend.

Reply:
I added it because it returns the password and is not used for auth. I have been trying to figure out the best approach with our limited roles. There are others than also return the password, but are needed without admin role. The only solution I can think of is either finer grained roles or splitting models based on need (a user model without password, for instance).

Reply:
Ah, OK makes sense. Apparently we can define a getter in the model like:

```
...
password: {
  type: DataTypes.STRING,
  get: () => null
},
...
```

And then you can use `model.getDataValue('password')` if you ever need access to the value.